doc/tools/west/index.rst

```diff
@@ -8,278 +8,46 @@ named ``west`` (Zephyr is an English name for the Latin
 `Zephyrus <https://en.wiktionary.org/wiki/Zephyrus>`_, the ancient Greek god
 of the west wind).
 
-West is developed in its own `repository on GitHub`_.
+West is used upstream to obtain the source code for the Zephyr project and can
```

Comment:
I am trying to make it clear that this is not the only usage. I don't think this sentence precludes downstreams from using it the same way. But without some qualifier, it sounds like a categorical statement that this is a requirement, which it's not. Can you think of a way to do that that would work better?

Reply:
How about something like:
```
West is developed to ease working on the Zephyr project and related forks.
West main features are:
- Multi-repository management
- Build, debug, and flash applications

West is designed in a modular way, so that it is easy to extend with new features
through loading of extension commands
```